library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -36,7 +37,17 @@
 
 	public final String currency;
 
-	public final Double priceValue;
+	public final double priceValue;
+
+	public final Period subscriptionPeriod;
+
+    public final Period subscriptionFreeTrialPeriod;
```

Comment:
What is the advantage of using this class from Joda Time? Just wanted to ask since you're adding a runtime dependency.